src/platform_impl/windows/dark_mode.rs

```diff
@@ -115,17 +122,17 @@ fn set_dark_mode_for_window(hwnd: HWND, is_dark_mode: bool) -> bool {
     if let Some(set_window_composition_attribute) = *SET_WINDOW_COMPOSITION_ATTRIBUTE {
         unsafe {
             // SetWindowCompositionAttribute needs a bigbool (i32), not bool.
-            let mut is_dark_mode_bigbool = is_dark_mode as BOOL;
+            let mut is_dark_mode_bigbool = BOOL::from(is_dark_mode);
 
             let mut data = WINDOWCOMPOSITIONATTRIBDATA {
                 Attrib: WCA_USEDARKMODECOLORS,
                 pvData: &mut is_dark_mode_bigbool as *mut _ as _,
                 cbData: std::mem::size_of_val(&is_dark_mode_bigbool) as _,
             };
 
-            let status = set_window_composition_attribute(hwnd, &mut data as *mut _);
+            let status = set_window_composition_attribute(hwnd, &mut data);
 
-            status != FALSE
+            status == true.into()
```

Comment:
Semantic change (positive comparison to `true` instead of negative comparison to `false`)